app/routes.py

```diff
@@ -63,3 +64,37 @@ def post_kml():
         ),
         201
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['GET'])
+def get_id(kml_admin_id):
+    logger.debug("entering route /kml/%s", kml_admin_id)
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    logger.debug(item)
+    # Fetching a non existing Item will return "None"
+    if item is None:
+        logger.info("exiting /kml/%s with a 400 status", kml_admin_id)
```

Comment:
Should be logged with ERROR level and a better message, e.g. `KML id %s has not been found in DB`

Reply:
You are right.